Package.swift

```diff
@@ -597,6 +597,7 @@ let package = Package(
       dependencies: [
         "FirebaseCore",
         "FirebaseInstallations",
+        "GoogleUtilities_Logger",
```

Comment:
This is out of date from master now, so you'll need to update before this can land.